mb_qol_inline_recording_tracks.user.js

```diff
@@ -101,9 +101,28 @@ function loadAndInsert() {
     });
 }
 
-// FIXME: onload workaround for hydration errors, not ideal, will cause script
-// to postpone initialising until cover art is loaded.
-window.addEventListener('load', () => {
+// MBS will fire a custom `mb-hydration` event whenever a React component gets
+// hydrated. We need to wait for hydration to complete before modifying the
+// component, React gets mad otherwise.
+// Multiple `mb-hydration` events will fire on a release page, so make sure we're
+// listening for the correct one.
+function onReactHydrated(element, callback) {
+    var alreadyHydrated = Object.keys(element).some(function (propertyName) {
+        return propertyName.indexOf('_reactListening') === 0 && element[propertyName];
```

Comment:
```suggestion
        return propertyName.startsWith('_reactListening') && element[propertyName];
```

Reply:
Yeah, copy-pasted this from https://github.com/metabrainz/musicbrainz-server/pull/2566#issuecomment-1155799353 where I intentionally didn't use more modern browser APIs. I'm not sure about browser support for `String#startsWith`, but it should already be polyfilled by MBS.